<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientInfoPop</class>
 <widget class="QDialog" name="ClientInfoPop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>客户代码（资金账号）*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_clientid"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>客户名称  *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_clientName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>客户类别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cm_cl_kind"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>开户日期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_firstdate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancle_btn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
